wiki/Gameplay/Slider_break/de.md

```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+tags:
+  - sliderbreak
+  - slider-break
+  - slider break
+  - combo break
+  - combobreak
+  - combo-break
+  - SB
+---
+
+# Sliderbreak
+
+**Sliderbreak** (manchmal abgekürzt als *SB*) ist ein Begriff, der verwendet wird, um das Unterbrechen einer Combo durch einen teilweise vollendeten [Slider](/wiki/Hit_object/Slider) zu beschreiben. Sliderbreaks können auf drei verschiedene Arten auftreten: den Kopf eines Sliders zu verfehlen, während der Körper noch ganz oder teilweise fertiggestellt ist, einen Slider zu irgendeinem Zeitpunkt seiner Dauer loszulassen oder nicht alle Ticks des Sliders zu sammeln. Alle drei Arten werden zusammen als "Sliderbreak" bezeichnet und beenden die Combo des Spielers. Ein Sliderbreak während eines Durchgangs führt zu einem "100er" und unterbricht die Combo, zählt aber nicht als Miss.
+
+*Hinweis: Ein Durchgang, der keine Misses, aber einen oder mehrere Sliderbreaks enthält, führt zu einer S-Note (oder Silber S), vorausgesetzt, dass alle [Genauigkeit](/wiki/Gameplay/Accuracy)-Anforderungen erfüllt wurden, um eine S-Note zu erhalten.*
```

Comment:
```suggestion
*Hinweis: Ein Durchlauf, der keine Misses, aber einen oder mehrere Sliderbreaks enthält, führt zu einer S-Note (oder Silber S), vorausgesetzt, dass alle [Genauigkeits](/wiki/Gameplay/Accuracy)-Anforderungen erfüllt wurden, um eine S-Note zu erhalten.*
```

Reply:
`Genauigkeitsanforderungen` might also be possible.

Reply:
i think this is good